pandas/tests/test_panel.py

```diff
@@ -2155,8 +2147,8 @@ def test_multiindex_get(self):
             assert (f1.items == [1, 2]).all()
             assert (f2.items == [1, 2]).all()
 
-            ind = MultiIndex.from_tuples([('a', 1), ('a', 2), ('b', 1)],
```

Comment:
what does this last section compare?

Reply:
Not entirely sure?  I just saw this variable being assigned and not being used afterwards.

Reply:
ok, if you can remove that (doesn't look like it does anything) would be great.

Reply:
Sure thing.  Done.